code/addons/test/src/vitest-plugin/test-utils.ts

```diff
@@ -22,10 +22,33 @@ export const testStory = (
 
     context.story = composedStory;
 
-    const _task = context.task as RunnerTask & { meta: TaskMeta & { storyId: string } };
+    const _task = context.task as RunnerTask & {
+      meta: TaskMeta & { storyId: string; instrumenterState: any };
+    };
     _task.meta.storyId = composedStory.id;
+    globalThis.__STORYBOOK_PREVIEW__ = {
```

Comment:
Ideally we would refactor the instrumenter to allow us to define the story id.
In PS there is no such thing as Storybook Preview, which is used by the instrumenter